frontend/views/containers/sidebar/Navigation.vue

```diff
@@ -46,23 +46,23 @@ nav.c-navigation(
           href='https://groupincome.org/blog/'
           target='_blank'
         )
-          i18n Blog
+          i18n(:class="isDarkTheme ? 'has-text-light-grey' : ''") Blog
 
         list-item(
           tag='a'
           variant='secondary'
           href='https://groupincome.org/faq/'
           target='_blank'
         )
-          i18n Help &amp; Feedback
+          i18n(:class="isDarkTheme ? 'has-text-light-grey' : ''") Help &amp; Feedback
```

Comment:
I believe this theme spec should not be addressed here but instead at `list-item`component itself so it's fixed everywhere it's used.